Preferences.sublime-settings

```diff
@@ -11,6 +11,14 @@
     {
       "selector": "source.c - string - comment - constant.numeric",
       "characters": ".>:",
+    },
+    {
+      "selector": "source.objc++ - string - comment - constant.numeric",
+      "characters": ".>: ",
```

Comment:
These were all necessary for me on my settings, including the space character.
If I remove the changes to this file and manually trigger completion (ctrl+space), I get the easy clang complete completions. But if I want the completions to automatically pop up as a type, I need this file.